labelbox/schema/organization.py

```diff
@@ -126,6 +127,23 @@ def remove_user(self, user: "User") -> None:
             updateUser(where: {id: $%s}, data: {deleted: true}) { id deleted }
         }""" % (user_id_param, user_id_param), {user_id_param: user.uid})
 
+    def get_resource_tags(self) -> List["ResourceTag"]:
+        """
+        Returns all resource tags for an organization
+        """
+        res = self.client.execute(
+            """query {
```

Comment:
You should probably name this query. Something descriptive like `getOrganizationResourceTagsPyApi` could work.